content/en/docs/topics/chart_repository.md

```diff
@@ -308,5 +308,5 @@ latest chart information.
 
 *Under the hood, the `helm repo add` and `helm repo update` commands are
 fetching the index.yaml file and storing them in the
-`$XDG_CACHE_HOME/helm/repository/cache/` directory. This is where the `helm
+`$(helm env HELM_CACHE_HOME)/repository` directory. This is where the `helm
```

Comment:
Helm is used in non-POSIX systems like Windows. Commands that generally target that Linux like environments are something we try to avoid. In #780 I changed the variable name to the Helm one. This isn't the best solution those environment variables may be empty and using the defaults.

Can we get a solution that works for both Windows and Linux/Mac? It could possible tell people who to get the value for the environment variables and then show where something is.